src/XMLInterface.cpp

```diff
@@ -518,4 +520,237 @@ void XMLInterface::writeXML(Scores& fullset, ProteinProbEstimator* protEstimator
   os << "</percolator_output>" << endl;
   os.close();
 }
+/* 
```

Comment:
You probably want to remove this comment.

Reply:
Fixed: https://github.com/percolator/percolator/commit/e56ae7dcaaa192c379958cf14dc9465b8c11b6c5